doc/source/tutorial/stats/hypothesis_pearsonr.md

```diff
@@ -0,0 +1,157 @@
+- Small p-values are not evidence for a large effect; rather, they can only provide evidence for a "significant" effect, meaning that they are unlikely to have occurred under the null hypothesis.
+
+Suppose that before performing the experiment, the authors had reason to predict a positive linear correlation between the total collagen and free proline measurements, and that they had chosen to assess the plausibility of the null hypothesis against a one-sided alternative: free proline has a positive linear correlation with total collagen. In this case, only those values in the null distribution that are as great or greater than the observed statistic would be considered more extreme.
+
+```{code-cell} ipython3
+res = stats.pearsonr(x, y, alternative='greater')
+res.statistic
+fig, ax = plt.subplots(figsize=(8, 5))
+plot(ax)
+pvalue = dist.sf(rs)
+annotation = (f'p-value={pvalue:.6f}\n(shaded area)')
+props = dict(facecolor='black', width=1, headwidth=5, headlength=8)
+_ = ax.annotate(annotation, (rs, 0.01), (0.86, 0.04), arrowprops=props)
+i = r_vals >= rs
+ax.fill_between(r_vals[i], y1=0, y2=pdf[i], color='C0')
+ax.set_xlim(0.85, 1)
+ax.set_ylim(0, 0.1)
+plt.show()
+```
+
+```{code-cell} ipython3
+res.pvalue  # one-sided p-value; half of the two-sided p-value
+```
+
+Note that the beta distribution is the exact null distribution for samples of any size under the null hypothesis that the samples were drawn from a normal distribution. We can convince ourselves of this by computing a Monte Carlo null distribution: explicitly drawing samples from independent normal distributions and computing Pearson's statistic for each pair.
+
+```{code-cell} ipython3
+def statistic(x, y, axis):
+    return stats.pearsonr(x, y, axis=axis).statistic  # ignore pvalue
+    
+ref = stats.monte_carlo_test((x, y), rvs=(rng.standard_normal, rng.standard_normal),
+                             statistic=statistic, alternative='greater', n_resamples=9999)
+
+fig, ax = plt.subplots(figsize=(8, 5))
+plot(ax)
+ax.hist(ref.null_distribution, np.linspace(-1, 1, 26), density=True)
+ax.legend(['exact null distribution (independent, normally-distributed observations)',
+           f'Monte Carlo null distribution \n({len(ref.null_distribution)} permutations)'])
+plt.show()
+```
+
+This is often an reasonable null hypothesis to test, but in other cases, it may be more conservative to perform a permutation test: Under the null hypothesis that total collagen and free proline are independent (but not necessarily normally distributed), each of the free proline measurements were equally likely to have been observed with any of the total collagen measurements. Therefore, we can form an exact null distribution by calculating the statistic under each possible pairing of elements between `x` and `y`. This is the null distribution used when we provide `pearsonr` with `method=stats.PermutationMethod()`.
```

Comment:
```suggestion
This is often an reasonable null hypothesis to test, but in other cases, it may be more conservative to perform a permutation test: Under the null hypothesis that total collagen and free proline are independent (but not necessarily normally distributed), each of the free proline measurements are equally likely to be observed with any of the total collagen measurements. Therefore, we can form an exact null distribution by calculating the statistic under each possible pairing of elements between `x` and `y`. This is the null distribution used when we provide `pearsonr` with `method=stats.PermutationMethod()`.
```